Library/ViewModels/SearchProjectCellViewModel.swift

```diff
@@ -0,0 +1,64 @@
+import KsApi
+import Prelude
+import ReactiveSwift
+import Result
+
+public protocol SearchProjectCellViewModelInputs {
```

Comment:
the cells for the "most popular" cell and the other cells seem to have the same data now, so I think  we could just use a single view model to service them both.